app/subscriber/src/store/slices/content/contentSlice.ts

```diff
@@ -34,23 +33,6 @@ export const initialContentState: IContentState = {
     logicalOperator: LogicalOperator.Contains,
     searchTerm: '',
   },
-  morningReportFilter: {
```

Comment:
@jdtoombs you didn't appear to be using this filter in the Subscriber site, so I removed it.